version.go

```diff
@@ -1,12 +1,49 @@
 package ftl
 
-import "regexp"
+import (
+	"regexp"
+	"strconv"
+)
+
+const VersionNumberParts int = 3
 
 // IsRelease returns true if the version is a release version.
 func IsRelease(v string) bool {
 	return regexp.MustCompile(`^\d+\.\d+\.\d+$`).MatchString(v)
 }
 
+// IsVersionAtLeastMin returns true if any of the following are true:
+//   - minVersion is not defined (i.e. is emptystring)
+//   - v or minVersion is not a release version
+//   - v > minVersion when both v and minVersion are release versions
+func IsVersionAtLeastMin(v string, minVersion string) (bool, error) {
```

Comment:
We're already importing `golang.org/x/mod/semver`, so I would just use that here rather than rolling our own.

Reply:
Oh awesome! Switched to using that :D